### PR TITLE
Add commands to move the review cursor to the first and last character of selected text

### LINKED ISSUE #17299: gesture to move to start/end of selection in documents and edit controls
### Is your feature request related to a problem? Please describe.
Hopefully just to improve ux. When selecting several lines of text you might want to quickly check where your selection starts. This is now possible with "announce current selection" but may require several key presses.
### Describe the solution you'd like
This should be easy by setting review position to start/end of selection. I think it would be consistent to have both selection start/end gestures.

### PR DESCRIPTION
**Pull request by nvdaes**
### Link to issue number:
Fixes #17299
### Summary of the issue:
NVDA doesn't have commands to move the review cursor to start and end of selection, which may be useful to review the selected text.
### Description of user facing changes
`NVDA+alt+home` and `NVDA+alt+end` can be used to move the review cursor to the first and last carácter of selected text.
### Description of development approach
In globalCommands.py, GlobalCommands class, a staticmethod has been added to get the current selection. This is used to scripts added to move the review cursor to start and end of selection, similar to other scripts used to move the review cursor.
### Testing strategy:
Tested manually in a webpage and in Notepad.
### Known issues with pull request:
None.
### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] API is compatible with existing add-ons.
- [x] Security precautions taken.

@coderabbitai summary

### CODE REVIEW

**Comment by nvdaes** [comment]
@burmancomp , please, feel free to test again in MS Word with UIA enabled only when needed or with other configurations and applications. I think that now this is fixed, according to my tests. Now we move the end of the selection info one character back before collapsing, instead of collapsing and then moving back one character.

**Comment by nvdaes** [comment]
Then let's mark this as ready for review.

File: user_docs/en/userGuide.md [hunk 1]
```diff
@@ -843,6 +843,8 @@ The following commands are available for reviewing text:
 |Move to end of line in review |shift+numpad3 |NVDA+end |none |Moves the review cursor to the end of the current line of text|
 |Move to previous page in review |`NVDA+pageUp` |`NVDA+shift+pageUp` |none |Moves the review cursor to the previous page of text if supported by the application|
 |Move to next page in review |`NVDA+pageDown` |`NVDA+shift+pageDown` |none |Moves the review cursor to the next page of text if supported by the application|
+|Move to start of selection |NVDA+alt+home |NVDA+alt+home |none |Moves the review cursor to the first character of the selected text|
+|Move to end of selection |NVDA+alt+end |NVDA+alt+end |none |Moves the review cursor to the last character of the selected text|
```
Comment (SaschaCowley) [inline user_docs/en/userGuide.md:847]: Can we include that these are review cursor gestures in the descriptions?
```suggestion
|Move to start of selection in review |NVDA+alt+home |NVDA+alt+home |none |Moves the review cursor to the first character of the selected text|
|Move to end of selection in review |NVDA+alt+end |NVDA+alt+end |none |Moves the review cursor to the last character of the selected text|
```

**Review by SaschaCowley — CHANGES_REQUESTED** [review]
Thanks @nvdaes. Mostly cosmetic changes requested, but I think you have accidentally deleted the move to end of line in review gesture.

File: source/globalCommands.py
[the remark this answers is not in the document]
Comment (nvdaes) [reply source/globalCommands.py:1]: Done

**Comment by nvdaes** [comment]
@SaschaCowley , think that all your suggestions are addressed now.
Thanks for your review, and thanks all reviewers.

**Review by SaschaCowley — APPROVED** [review]
Thanks @nvdaes

**Pull request merged**